tests/scenarios/core-resolver-test.ts

```diff
@@ -384,6 +430,70 @@ Scenarios.fromProject(() => new Project())
           pairModule.resolves('../../component.js').to('./pods/components/hello-world/component.js');
         });
 
+        test('nested layout (in app) component resolved', async function () {
+          givenFiles({
+            'components/local-nested-layout-component/index.js': `
+              import { hbs } from 'ember-cli-htmlbars';
+              import { setComponentTemplate } from '@ember/component';
+              import templateOnly from '@ember/component/template-only';
+              const TEMPLATE = hbs('<div>{{this.message}}</div>')
+              setComponentTemplate(TEMPLATE, templateOnly());
+            `,
+            'app.js': 'import "#embroider_compat/components/local-nested-layout-component"',
+          });
+
+          await configure({
+            // options: {
```

Comment:
I saw in the other PR that the tests were using static settings, but they're not yet relevant here, else the tests would pass!